app/views/frontend/conferences/show.html.haml

```diff
@@ -13,11 +13,11 @@
       %h2= @conference.acronym.gsub(/[\-_]/,' ')
 
     #sorting.btn-group.btn-group-sm.btn-group-justified
-      - %w{name duration date}.each do |sorting|
+      - %w{name duration date view_count}.each do |sorting|
         - if @sorting == sorting
-          %a.btn.btn-primary.active{ href: conference_path(acronym: @conference.acronym, sort: sorting) }= sorting
+          %a.btn.btn-primary.active{ href: conference_path(acronym: @conference.acronym, sort: sorting) }= sorting.gsub(/_/, ' ')
```

Comment:
I had just a local dict or sth. in mind, but hey, I'll merge it, refactoring can be done later and this ha been long enough.